packages/integration-tests/src/__tests__/integration/mssql/__snapshots__/introspection.test.ts.snap

```diff
@@ -818,37 +797,17 @@ datasource sqlserver {
 }
 
 model users {
-  id    Int    @id @default(autoincrement())
+  id    Int    @id(map: "PK__users__CustomNameToAvoidRandomNumber") @default(autoincrement())
   email String @sqlserver.VarChar(32)
   name  String @sqlserver.VarChar(32)
 
-  @@unique([email, name], name: "u_const")
+  @@unique([email, name], map: "u_const")
 }
 
 `;
 
 exports[`introspection: findUnique where composite unique: warnings 1`] = `Array []`;
 
-exports[`introspection: findUnique where decimal - default value: datamodel 1`] = `
-generator client {
-  provider = "prisma-client-js"
-  output = "***"
-}
-
-datasource sqlserver {
-  provider = "sqlserver"
-  url = "***"
-}
-
-model exercises {
-  id       Int     @id @default(autoincrement())
-  distance Decimal @unique @default(12.3) @sqlserver.Decimal(5, 3)
-}
-
-`;
-
-exports[`introspection: findUnique where decimal - default value: warnings 1`] = `Array []`;
```

Comment:
huh removed?

Reply:
Nice catch, Pr up at https://github.com/prisma/prisma/pull/9067

Reply:
How did this not make the tests fail here when you remove a snapshot? I am confused.

Reply:
Indeed